crates/experimental-api/tests/exe.rs

```diff
@@ -0,0 +1,41 @@
+use beerus_experimental_api::{
+    exe::exec,
+    gen::{client::blocking::Client, FunctionCall},
+};
+
+#[allow(dead_code)]
```

Comment:
Is this necessary?

Reply:
It just keeps slipping in :) Thank you for catching it!

Reply:
Done.